src/simulator/base.py

```diff
@@ -0,0 +1,27 @@
+from object import Object, ObjectId
+from position import Position
+from shape import Shape
+from object_registry import ObjectRegistry
```

Comment:
This might cause the circular import problem since in Object.py, ObjectRegistry is imported and vice versa.
Maybe it's better to use the format Erfan suggested for resolving circular import, i.e:
```
import object
import object_registry
```
and then use this modules as `object.Object` and `object_registry.ObjectRegistry`

Reply:
I didn't get in. did you mean just like what I pushed now?

Reply:
Yes, Exactly!